docs/Makefile

```diff
@@ -1,61 +1,56 @@
 HUGO?=hugo
-HUGO_IMG?=hugomods/hugo:0.134.1
+HUGO_VERSION?=$(shell hugo version 2>/dev/null | awk '{print $$2}' | cut -d '.' -f 2)
+HUGO_IMG?=hugomods/hugo:std-go-git-0.134.3
 
 THEME_MODULE = github.com/nginxinc/nginx-hugo-theme
-THEME_VERSION ?= $(shell curl -s https://api.github.com/repos/nginxinc/nginx-hugo-theme/releases/latest | jq -r ".tag_name")
 
-ifeq (, $(shell ${HUGO} version 2> /dev/null))
-ifeq (, $(shell docker version 2> /dev/null))
-    $(error Docker and Hugo are not installed. Hugo (<0.91) or Docker are required to build the local preview.)
+ifeq ($(shell [ $(HUGO_VERSION) -gt 133 2>/dev/null ] && echo true || echo false), true)
+    $(info Hugo is available and has a version greater than 133. Proceeding with build.)
 else
-    HUGO=docker run --rm -it -v ${CURDIR}:/src -p 1313:1313 ${HUGO_IMG} hugo --bind 0.0.0.0 -p 1313
-endif
+    $(warning Hugo is not available or using a version less than 134. Attempting to use docker. HUGO_VERSION=$(HUGO_VERSION))
+    HUGO=docker run --rm -it -v ${CURDIR}:/src -p 1313:1313 ${HUGO_IMG} /src/hugo-entrypoint.sh
+    ifeq (, $(shell docker version 2> /dev/null))
+        $(error Hugo (>0.134) or Docker are required to build the local previews.)
+    endif
 endif
 
-HUGO_CI=docker run --rm -v ${CURDIR}:/src ${HUGO_IMG} hugo
```

Comment:
we use this in the CI, `make docs` which uses the `HUGO` variable doesn't work in the CI, because of it `-it` I think

Reply:
https://github.com/nginxinc/kubernetes-ingress/blob/76651cc80eec84fa7cc7ffd29c2733996e1ee663/.github/workflows/ci.yml#L207

Reply:
Ah I didn't spot that. Thanks, will update 👍